src/Pages/Home/Home.css

```diff
@@ -0,0 +1,19 @@
+.chip-section{
+    padding : 1rem;
+}
+
+.searchbar-section{
+    padding : 1rem;
+}
+
+.video-listing-section{
+    padding : 1rem;
```

Comment:
you can create one paddding class instead of giving same padding in different classes